optional-deps/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "optional-deps"
+version = "0.1.0"
+authors = [
+    "Eh2406 <YeomanYaacov@gmail.com>",
+    "Matthieu Pizenberg <matthieu.pizenberg@gmail.com>",
+]
```

Comment:
The `authors` field is now optional, we can remove it if you want.

Reply:
I don't mind either way but let's do it so we don't have to update it later with new authors XD